### ISSUE TITLE
Bug: Multiple ignore params get clobbered by the last one

### ISSUE THREAD

**Issue by willsmanley-uber** [alexcanessa/typescript-coverage-report#118]
**Describe the bug**
In type-coverage core, you can pass multiple ignore params and they will all be honored. In this wrapper, only the last ignore param is honored.

**To Reproduce**
Use two separate ignore params. Only the last one will be ignored.

**Expected behaviour**
Both file globs should be ignored.

**Comment by willsmanley-uber** [comment]
@alexcanessa FYI, just wanted to get your thoughts here

**Comment by wolverineks** [comment]
@willsmanley-uber i was able to get it to work using package.json options
https://github.com/plantain-00/type-coverage#config-in-packagejson

**Comment by luisrudge** [comment]
yeah, I'm having this issue as well when using the CLI. I can't use the package.json option because I have multiple package.json files in my repo.

**Comment by alexcanessa** [comment]
Ok thanks @luisrudge.
Not sure if you'd be keen to contribute to the repo? Otherwise I'll see what I can do in the weekend.

**Comment by alexcanessa** [comment]
**2.0.0 is live**, and repeated `-i` now works as the help text always claimed:

```shell
typescript-coverage-report -i "demo1/*.ts" -i "demo2/foo.ts"
```

@willsmanley-uber @luisrudge @wolverineks — worth knowing this changes your numbers if you were passing `-i` more than once. Previously only the last occurrence applied, so your reports will now ignore more than before.

@luisrudge your monorepo case should work now without needing the package.json config.
